tests/ert3/console/integration/test_cli.py

```diff
@@ -252,11 +252,55 @@ def _assert_export(workspace, experiment_name):
     _assert_parameter_statistics(config, export_data)
 
 
-def test_cli_export_polynomial_evaluation(tmpdir):
+def _find(keypath, content):
+    keys = keypath.split(".")
+    elem = content
+    for key in keys:
+        try:
+            key = int(key)
+        except ValueError:
+            pass
+        elem = elem[key]
+    return elem
+
+
+def _update_content(fname, updates):
+    """
+    Takes a filename that is expected to be yaml formatted,
+    and updates as a list of tuples of keypath and value,
+    where keypath is on the format key_name1.key_name2 etc.
+
+    Note that the key_name will be resolved as an int if possible,
+    and as such it is also possible to search list-like structures
+
+    Will update each keypath with given value and dump file to disk
+    """
+    with open(fname) as f:
+        content = yaml.safe_load(f)
+    for keypath, value in updates:
+        elem = _find(keypath, content)
+        elem = value
```

Comment:
I don't think this work as you intended... ;)

Reply:
I must admit I didn't like how this looked, and I also thought this would just reassign the element. Somehow, in my head, I thought I was passing the reference and reassigning it. The reason this is here at all is that when running the tests it looked to be changing the content of the configuration. I'll make this properly though (need a test for test utilities it looks like :P )

Reply:
Updated, but I wonder if there's a better way of handling this?

Not particularly fond of the logic necessary to make this, would it be better to add the entire config in the parametrize section?